src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static addAnimal(animal) {
+    Animal.alive.push(animal);
+  }
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
```
  constructor(name, health = 100, hidden = false) {
    this.name = name;
    this.health = health;
    this.hidden = hidden;
```

Reply:
Agree with you.. Your solution is more understandable!